src/code/qualification/plotQualificationGOFMerged.m

```diff
@@ -109,7 +109,7 @@
 
 for i=1:length(Groups)
     
-    for j=1:length(Groups(i))
+    for j=1:length(Groups(i).OutputMappings)

```

Comment:
This was clearly wrong. After the change it crashes in the line 
121 `Error = [Error log10(Group(i).dataTP(j).ypred)-log10(Group(i).dataTP(j).yobs)];`
no idea how to fix that - will create a separate issue

Reply:
Issue: #153 
@pchelle Please fix asap